### PR TITLE
Sign in to phantom

### PR DESCRIPTION
**Pull request by niyontwali**
#### Main object of the Task
The driver and operator should be able to login ith credentials provided by the admin, so that they can perform their tasks. 
#### Description of Task to be completed
1. The driver, admin and operator should be able to sign in to the app by providing their email & password
2. The API should return a JW toke on a successful login
3. The administror should be able to sign in to the app using pre-configured login credentials
4. Appropriate error messages are displayed on unsuccessful attempts
#### How should this be manually tested
1. First and foremost clone the repo, cd into it and `run npm run seed` to help you create the dummy data in the seeder folder
2. You can run tests by running `npm run test` in your terminal
3. Run the server by running `npm run dev` in your terminal
4. Using Postman test the POST: login endpoint (`http://localhost:3000/users/login`) which by sending the email and password in the user_dummy data. 
_key:_Note that the postman body must be raw and json content. After a successful login you will see that a token will be generated.
#### Screenshots 
Successful Login
Unsuccessful login
User's email does exist

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+
+- run migration to create database
+
+- npx sequelize-cli  db:migrate
+<<<<<<< HEAD
+=======
+- npx sequelize-cli db:seed:all
+```
+>>>>>>> afe8fbb (setup project postgres)
+
```
Comment (SimonImanigirimpuhwe) [inline README.md:29]: What is this? You didn't rebase

File: README.md [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+## Installation
+
+- `git pull this branch`
+- `npm install`
+- run `npm run test` you should see test passing
+
+![runtest](https://user-images.githubusercontent.com/50244289/155469886-bff4a8d6-4af9-47dc-8bec-efa0a48a1500.png)
+=======
+>>>>>>> afe8fbb (setup project postgres)
```
Comment (SimonImanigirimpuhwe) [inline README.md:67]: Again

File: src/controllers/login.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+const login = ('/login', (req, res) => {
+  const { email, password } = req.body
+  User.findOne({
+    where: {
+      email
+    }
+  }).then(user => {
+    if (!user) {
+      res.status(400).json({
+        error: 'The email is not registered! Please first register'
```
Comment (SimonImanigirimpuhwe) [inline src/controllers/login.js:17]: Make sure you're supporting multiple languages

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -34,9 +34,21 @@ jobs:
...
+=======
+          heroku_api_key: ${{secrets.b22f31da-9e25-401e-b066-b96bc29c3819}}
+          heroku_app_name: 'phantom-api-pirates'
+          heroku_email: 'habchris15@gmail.com'
+>>>>>>> d6e25fa (configure CI/CD using Github Actions and Heroku)
+=======
+          heroku_api_key: ${{secrets.HEROKU_KEY}}
+          heroku_app_name: HEROKU_APP_NAME
+          heroku_email: HEROKU_EMAIL
+>>>>>>> d25f07f (github action secret keys added)
```
Comment (SimonImanigirimpuhwe) [inline .github/workflows/main.yml:51]: Again, re-review your rebasing.

**Review by SimonImanigirimpuhwe — CHANGES_REQUESTED** [review]
Work on that and consider squashing your commits, we can't merge a single feature with 69 commits.

File: src/app.js [hunk 5]
```diff
@@ -1,20 +1,23 @@
 import express from 'express'
 import logger from 'morgan'
+import { sequelize } from '../models/index'
 import swaggerJsDoc from 'swagger-jsdoc'
 import swaggerUI from 'swagger-ui-express'
-import bodyParser from 'body-parser'
```
Comment (christianR15c) [inline src/app.js:5]: I think body-parser is needed, should not have been removed

File: src/app.js [hunk 6]
```diff
@@ -28,9 +31,8 @@ app.all('*', function (req, res, next) {
   next()
 })
 
-app.use(bodyParser.json({ limit: '100mb' }))
-app.use(bodyParser.urlencoded({ limit: '50mb', extended: 'true' }))
-app.use(bodyParser.json({ type: 'application/vnd.api+json' }))
+app.use(express.json())
```
Comment (christianR15c) [inline src/app.js:34]: consider having these codes back in your codes

**Review by christianR15c — CHANGES_REQUESTED** [review]

**Pull request closed without merging**